pallets/phala/src/migrations/mod.rs

```diff
@@ -120,3 +140,263 @@ pub mod v6 {
 		Ok(())
 	}
 }
+
+pub mod stakepoolv2_migration {
+	use super::*;
+
+	const STAKING_ID: LockIdentifier = *b"phala/sp";
+	const VESTING_ID: LockIdentifier = *b"vesting ";
+	const DEMOCRAC_ID: LockIdentifier = *b"democrac";
+	const PHRELECT_ID: LockIdentifier = *b"phrelect";
+	fn migrate_stake_pools<T: PhalaPallets>()
+	where
+		T: PhalaPallets + stakepool::pallet::Config + PhalaConfig,
+		BalanceOf<T>: balance_convert::FixedPointConvert + sp_std::fmt::Display,
+		T: pallet_uniques::Config<CollectionId = CollectionId, ItemId = NftId>,
+		T: pallet_assets::Config<AssetId = u32>,
+		T: pallet_assets::Config<Balance = BalanceOf<T>>,
+	{
+		stakepool::pallet::StakePools::<T>::iter().for_each(|(pid, pool_info)| {
```

Comment:
Drain it?

Reply:
We can not drain it because we need to caculate stakers rewards in the latter function.

Reply:
Drain it in the last step

Reply:
done